ibis-server/app/config.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from dotenv import load_dotenv
+
+
+class Config:
+    def __init__(self):
+        load_dotenv(override=True)
+        self.wren_engine_endpoint = os.getenv('WREN_ENGINE_ENDPOINT')
```

Comment:
Seems the env is required. If have a check when server starting, it's would be better.